en/docs/monitoring-and-insights/integrate-choreo-with-moesif.md

```diff
@@ -0,0 +1,85 @@
+# Integrate Choreo with Moesif
```

Comment:
Since this page is under "Monitoring and Insights" just wondering whether it will be better to have a title similar to "Monitor API Insights via Moesif"

Reply:
Moesif not only gives API analytic features, but also monetization. I thought that someone who knows about Moesif will know what this is about.

Reply:
Ack.